ddemangle.d

```diff
@@ -58,10 +60,32 @@ const(char)[] demangleMatch(T)(Captures!(T) m)
     }
 }
 
-auto ddemangle(T)(T line)
-    if (is(T : const(char)[]))
+const(char)[] demangleMatchUnderscoreMissing(T)(Captures!(T) m)
+        if (is(T : const(char)[]))
 {
-    return replaceAll!(demangleMatch)(line, reDemangle);
+    static Appender!string ret;
```

Comment:
Why is this `static`?

Reply:
so that subsequent invocations don't reallocate; do you have any better suggestion?

Reply:
Use `chain` and `choose` for zero allocation?

Reply:
demangle accepts `const(char)[] buf` how could chain work?